src/map/entities/charentity.cpp

```diff
@@ -364,7 +364,7 @@ CCharEntity::~CCharEntity()
     destroy(Container);
     destroy(UContainer);
     destroy(CraftContainer);
-    destroy(PMeritPoints);
+    PMeritPoints.reset();
```

Comment:
This isn't needed. The point of using `unique_ptr` is that it'll clear up the underlying object when it is destructed, which is in this scope.